venues/cv-foundation.org/ECCV/2018/Conference/python/setup-matching.py

```diff
@@ -35,6 +35,28 @@
     }
 }))
 
+metadata_reviewers_inv = client.post_invitation(openreview.Invitation(**{
+    'id': 'cv-foundation.org/ECCV/2018/Conference/-/Paper_Metadata/Reviewers',
```

Comment:
Why not call the group like this:

cv-foundation.org/ECCV/2018/Conference/Reviewers/-/Paper_Metadata?

Reply:
good idea. that would work